src/main/java/de/srendi/advancedperipherals/common/util/ChunkManager.java

```diff
@@ -65,12 +66,10 @@ public static void afterServerStarted(ServerStartedEvent event) {
     }
 
     @SubscribeEvent
-    public static void serverTick(TickEvent.ServerTickEvent event) {
-        if (event.phase == TickEvent.Phase.END) {
+    public static void serverTick(ServerTickEvent.Pre event) {
+        if (event.hasTime()) {
             tickCounter++;
-            // run cleanup per chunkLoadValidTime / 10
-            final int checkIntervalInTick = APConfig.PERIPHERALS_CONFIG.chunkLoadValidTime.get() * 20 / 10;
-            if (tickCounter % checkIntervalInTick == 0) {
+            if (tickCounter % (APConfig.PERIPHERALS_CONFIG.chunkLoadValidTime.get() / 2) == 0) {
```

Comment:
why / 2 tho?

Reply:
That was actually not me, but from #677 
@diemoell can maybe answer that

My old crusty eyes didn't spot that while reviewing

Reply:
Reverted to the old formula